src/AWS.Deploy.Orchestration/ServiceHandlers/AWSElasticBeanstalkHandler.cs

```diff
@@ -26,6 +26,14 @@ public interface IElasticBeanstalkHandler
         /// The two main settings that are updated are IIS Website and IIS App Path.
         /// </summary>
         void SetupWindowsDeploymentManifest(Recommendation recommendation, string dotnetZipFilePath);
+
+        /// <summary>
+        /// When deploy a self contained deployment bundle Beanstalk needs a Procfile to tell the environment what process to start up.
```

Comment:
replace "deploy" by "deploying" and add a comma after "bundle"

Reply:
Done